api/controllers/incidents_controller.py

```diff
@@ -0,0 +1,158 @@
+from api.models.incidents_model import Incident
+from flask import jsonify, request
+from api.validation import Validating_string
+from db import Database_connection
+import psycopg2
+
+incident_obj = Incident()
+database_conn = Database_connection()
+class IncidentsController:
+
+    @staticmethod
+    def create_incidence():
+        data = request.get_json()
+        incident_type = data.get("incident_type")
+        location = data.get("location")
+        comment = data.get("comment")
+        user_id = data.get("user_id")
+
+        incidents_details = [incident_type, location, comment, user_id]
+        for incident in incidents_details:
+            if type(incident) is str:
+                if not Validating_string.characters(incident) or\
+                Validating_string.is_space(incident):
+                    return jsonify({
+                        "status": 400,
+                        "message": "All fields must be filled!"
```

Comment:
Can't we have a validation for this so that we can tell the user which field is empty